<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivitySummary</class>
 <widget class="QMainWindow" name="ActivitySummary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="UpperLogoBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="LogoStack_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <widget class="QTextEdit" name="Motto_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>914</width>
          <height>80</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-bottom:3px;
color:white;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ACCOUNTING MADE EASY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="MainWindow">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>300</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset>
          <activeon>:/images/AppLogo.png</activeon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <zorder>MainWindow</zorder>
       <zorder>Motto_2</zorder>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;

font: 900 10pt &quot;Segoe UI Black&quot;;
</string>
      </property>
      <property name="text">
       <string>Account Activity Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>180</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>113</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Time Period</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>From</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>To</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QDateEdit" name="ReadFromDate">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>60</y>
            <width>110</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="ReadToDate">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>110</y>
            <width>110</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>150</y>
            <width>80</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10;
background-color:gray;</string>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>433</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>150</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>400</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Beginning Balance</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="ReadOpeningBalance">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>20</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;
color:white;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Ending Balance</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="ReadClosingBalance">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>80</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:gray;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="GoBack">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>150</y>
            <width>80</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10;
background-color:gray;</string>
          </property>
          <property name="text">
           <string>Go BACK</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="LabelAccount">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;
font: 900 9pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Here is the Activity for selected account</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="ActivityTable">
      <property name="minimumSize">
       <size>
        <width>1100</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    background-color: rgb(194,196,188);     /* Background color of the table */
    gridline-color: #8F8F8F;        /* Gridline color */
    border: 2px solid #A0A0A0;      /* Border around the table */
}

QHeaderView::section {
    background-color: rgb(91,97,97); 
color:white;     /* Header background color */
    padding: 8px;                   /* Header padding */
    font-weight: bold;              /* Header font */
    border: 2px solid #A0A0A0;      /* Border around headers */
}

QTableWidget::item {
    padding: 5px;                   /* Padding for cells */
    background-color: rgb(209,212,203);      /* Default background color for cells */
}

QTableWidget::item:alternate {
    background-color: rgb(233,245,201);      /* Alternating row color */
}

QTableWidget::item:selected {
    background-color: #A0A0FF;      /* Background color when a cell is selected */
    color: white;                   /* Text color when a cell is selected */
}

QTableWidget QScrollBar:vertical {
    background: #F5F5F5;            /* Vertical scrollbar background */
    width: 15px;                    /* Scrollbar width */
}

QTableWidget QScrollBar::handle:vertical {
    background: #B0B0B0;            /* Vertical scrollbar handle color */
    border-radius: 4px;             /* Rounded scrollbar handle */
}
</string>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>220</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Transaction Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Journal Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>BY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Beginning Balance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ending Balance</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1196</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
